logo.js

```diff
@@ -1600,10 +1738,12 @@ function LogoInterpreter(turtle, stream, savehook)
   def("true", function() { return 1; });
   def("false", function() { return 0; });
 
+// FIXME: needs to handle promises
```

Comment:
I guess I'd only expect special forms to ever see promises. Maybe higher-order procs - I haven't thought it all the way through yet.